extensions/amp/bundle/src/main/java/com/adobe/cq/wcm/core/extensions/amp/internal/AmpModeForwardFilter.java

```diff
@@ -116,4 +125,20 @@ public void init(final FilterConfig config) {}
      */
     @Override
     public void destroy() {}
+    
+    
+    @Activate
```

Comment:
why not using constructor injection with overloaded constructors (one taking Config, calling the other one taking just the boolean flag). That eases testing a lot.

Reply:
done